src/main/java/bio/terra/cli/command/user/sshkey/Generate.java

```diff
@@ -22,8 +23,15 @@ public class Generate extends BaseCommand {
       description = "Save the terra ssh key pair as file, skip printing out the key")
   boolean saveToFile;
 
+  @CommandLine.Mixin ConfirmationPrompt confirmationPromptOption;
+
   @Override
   protected void execute() {
+    confirmationPromptOption.confirmOrThrow(
```

Comment:
Would be nice if we only showed this when there was a key...

Reply:
Maybe not worth the extra complexity though...

Reply:
agree. i don't think it's worth another round trip